src/app/CommandHandler.cpp

```diff
@@ -127,7 +128,15 @@ CHIP_ERROR CommandHandler::ProcessInvokeRequest(System::PacketBufferHandle && pa
         VerifyOrReturnError(TLV::AnonymousTag() == invokeRequestsReader.GetTag(), CHIP_ERROR_INVALID_TLV_TAG);
         CommandDataIB::Parser commandData;
         ReturnErrorOnFailure(commandData.Init(invokeRequestsReader));
-        ReturnErrorOnFailure(ProcessCommandDataIB(commandData));
+
+        if (mpExchangeCtx != nullptr && mpExchangeCtx->IsGroupExchangeContext())
```

Comment:
When would `mpExchangeCtx` be null?  Is this unit tests that test in a weird setup?  If so, worth adding a TODO comment explaining that and filing an issue on the tests.

Reply:
all the nullptr checks are an old habit. removed the check